### PR TITLE
[Chef-17] 12 of X - Correcting Windows Verify Plan failures

### PR DESCRIPTION
**Pull request by johnmccrae**
## Description
Neha discovered unneeded and unused rpm/yumrepo files that were causing the verify-plan tests to fail on Windows. https://github.com/chef/chef/pull/13411 We add those changes here to get Chef-17 past the same problem set.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Review by neha-p6 — CHANGES_REQUESTED** [review]
@johnmccrae You have added an entirely new directory as `/spec/functional/assets/yumrepo-empty/` and all the `.bz2` files under it. They are **not** there on chef-17 branch. That's why the windows plan is (still) failing and has more failures.
I think you copied over those from this pr of mine https://github.com/chef/chef/pull/13529/files#diff-135ca81b796cceed1277a3e4ae29cd5010a956690e6e1ccc3d4b2f9f07b402bd
That is a new complete new PR (rather a backport of new feature to chef16)

You will want to rename (shorten the file names) these files here https://github.com/chef/chef/pull/13411/files#diff-86ae48aa5bbe5ea7f9f979904a8d29948d2137aa36cd509aa5fc16c7687643eb
and update their corresponding references in repomd.xml (Note that I have removed those files in there since they are not used, but your could shorten their names as well, that's the root cause, the long file names)

File: habitat/plan.ps1 [hunk 1]
```diff
@@ -1,3 +1,8 @@
+
+# debugging - jfm
+$ErrorActionPreference = 'Continue'
```
Comment (neha-p6) [inline habitat/plan.ps1:3]: Could we remove this debugging change now?

**Review by neha-p6 — CHANGES_REQUESTED** [review]
@johnmccrae I think you need to this base this branch with latest changes from chef-17 branch. The windows plan bundle install issue now solve https://buildkite.com/chef-oss/chef-chef-chef-17-verify/builds/908#0187b3ed-1ba6-4742-a182-5462668424ea/1300-1475 and we can see hab package is now built successfully now that you have shortened the file names. It is now getting timed out when running functional tests.
Windows functional tests are fixed here https://github.com/chef/chef/pull/13709 but it doesn't seem those changes are here in this branch, so a rebase will fix it.

File: habitat/plan.ps1 (hunk 1 above)
Comment (johnmccrae) [reply habitat/plan.ps1]: Done

**Pull request merged**